### DISCUSSION TITLE
Force `@return` to match return type

### DISCUSSION THREAD

**Discussion by tillkruss** [PHP-CS-Fixer/PHP-CS-Fixer#6815] [Q&A]
## Support question

Is there a way to automatically fix phpdoc `@return` if they don't match the return type?

<img width="991" alt="Screenshot 2023-02-03 at 9 28 23 AM" src="https://user-images.githubusercontent.com/665029/216672630-c0174315-4f46-473b-8d38-b2ae8a268bf4.png">

**Comment by Wirone** [comment]
The best fix here is... removing redundant phpDoc 😉. Try `no_superfluous_phpdoc_tags` rule to get rid of duplicated information. Maybe you should apply it before changing native type (I don't remember if it can determine "superfluousness" if  native type is wider).
